addons/sale/tests/test_taxes_downpayment.py

```diff
@@ -756,3 +764,38 @@ def test_down_payment_analytic_distribution_aggregation(self):
             # Receivable line:
             {'tax_ids': [],                     'analytic_distribution': False,                                 'balance': 3375.0},
         ])
+
+    def test_downpayment_invoice_lines_with_down_payment_account(self):
+        sale_order = self.env['sale.order'].create({
+            'partner_id': self.partner_a.id,
+            'order_line': [
+                Command.create({
+                    'product_id': self.company_data['product_order_no'].id,
+                    'product_uom_qty': 1,
+                }),
+                Command.create({
+                    'product_id': self.company_data['product_order_sales_price'].id,
+                    'product_uom_qty': 2,
+                }),
+                Command.create({
+                    'product_id': self.company_data['product_order_cost'].id,
+                    'product_uom_qty': 3,
+                }),
+            ]
+        })
+        sale_order.action_confirm()
+        so_context = {
+            'active_model': 'sale.order',
+            'active_ids': [sale_order.id],
+            'active_id': sale_order.id,
+        }
```

Comment:
Do we need to use the context?
Can't we simply pass those values as parameters to the `create()` method, along with the payment params?